src/lib/preferences.ts

```diff
@@ -69,15 +68,11 @@ for (let k in defaults) {
 
 
 export function get(key: string): any {
-    return data.ensureDefaultAndGet(`preferences.${key}`);
```

Comment:
This has protected me more than once, I'd like to preserve the functionality. We can move the check here though and ditch the method in `data` if we want, I don't think we use it elsewhere.